sql/tables.sql

```diff
@@ -146,5 +146,35 @@ CREATE INDEX IF NOT EXISTS chunk_index_hypertable_id_hypertable_index_name_idx
 ON _timescaledb_catalog.chunk_index(hypertable_id, hypertable_index_name);
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.chunk_index', '');
 
+CREATE TABLE IF NOT EXISTS _timescaledb_catalog.bgw_job (
+    id                  SERIAL      PRIMARY KEY,
+    job_name            NAME        NOT NULL,
+    job_type            NAME        NOT NULL,
```

Comment:
I do not like the idea of using enum types since alter type add value cannot be executed inside a txn block and there is no alter type remove value at all (https://www.postgresql.org/docs/10/static/sql-altertype.html). This will make it hard to maintain in the future since we won't be able to use update scripts to change the set of values in an enum. We can consider creating a custom type where the in and out functions are defined in C and correspond to a C enum but I don't know if we want to go that far...